### DISCUSSION TITLE
Is tflint plugin supported as private plugin

### DISCUSSION THREAD

**Discussion by andylim0221** [terraform-linters/tflint#2145] [Q&A] [answered]
We are interested to develop plugin that are only open for internal usage. However, apparently tflint plugin can only be pulled from public repository. Is private plugin supported? If yes, what are the configuration that I need?

**Comment by bendrucker** [comment, marked as answer]
Yes, you just need to supply a GitHub auth token with access to read the repository releases. 

https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/plugins.md

**Reply by andylim0221** [reply]
Thank you for the documentation and it's helpful!
